library/src/main/java/com/novoda/downloadmanager/lib/OkHttpNotificationImageRetriever.java

```diff
@@ -36,4 +49,14 @@ public Bitmap retrieveImage(String imageUrl) {
             return null;
         }
     }
+
+    private void cleanupOldBitmap() {
+        if (bitmap == null) {
+            return;
+        }
+        if (!bitmap.isRecycled()) {
+            bitmap.recycle();
```

Comment:
as much as I like this, the [docs say](https://developer.android.com/reference/android/graphics/Bitmap.html#recycle())

> This is an advanced call, and normally need not be called, since the normal GC process will free up this memory when there are no more references to this bitmap.